nestkernel/conn_builder.h

```diff
@@ -144,7 +144,7 @@ class ConnBuilder
    * node is not located on the current thread or MPI-process and read of an
    * array.
    */
-  void skip_conn_parameter_( thread );
+  void skip_conn_parameter_( thread, int skip_num = 1 );
```

Comment:
Since the number of places to skip cannot be negative, I would suggest `size_t` as data type. I would also think `num_skip` or `n_skip` would be better names, closer to the full "number of positions to skip".